docs/guides/datasets/datasets-for-finetuning.mdx

```diff
@@ -0,0 +1,53 @@
+---
+title: Using Datasets for Fine-tuning
+description: Export Latitude datasets to prepare data for fine-tuning language models.
+---
+
+While Latitude focuses on prompt engineering and evaluation, the datasets you create and curate within the platform can be valuable assets for fine-tuning language models using external tools and services.
+
+## Why Use Latitude Datasets for Fine-tuning?
+
+- **Curated Data**: Datasets often contain carefully selected inputs and high-quality outputs (either expected outputs or actual model responses reviewed manually).
+- **Real-World Examples**: Datasets created from production logs represent actual user interactions.
+- **Structured Format**: Latitude datasets are already in a structured format (CSV), making them easier to process for fine-tuning.
+
+## Exporting Datasets from Latitude
+
+1.  Navigate to the "Datasets" section in your project.
+2.  Locate the dataset you want to use for fine-tuning.
+3.  Find the option to **Download** or **Export** the dataset (usually represented by a download icon).
```

Comment:
We don't have this feature. I think we never did

Reply:
removed this section